### ISSUE TITLE
[Performance][Request] Enable/Disable settings for ZenjectStateMachineBehaviourAutoInjecter

### ISSUE THREAD

**Issue by legoritma** [modesttree/Zenject#38]
Hi,
We're currently heavily use animators however not using injection in animations. Zenject checs that animator has ZenjectStateMachineBehaviourAutoInjecter and Add if not (in AddStateMachineBehaviourAutoInjectersUnderGameObject method).

Is it possible add options for turn-off this controls?

**Comment by svermeulen** [comment]
Yes we can add a setting to disable this behaviour in ZenjectSettings
